requirements-test.txt

```diff
@@ -0,0 +1,46 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile requirements-test.in
+#
+aniso8601==8.0.0          # via flask-restplus
+attrs==19.3.0             # via jsonschema, pytest
+certifi==2019.11.28       # via geventhttpclient-wheels, requests
+chardet==3.0.4            # via requests
+click==7.0                # via flask
+daiquiri==1.6.0
+flask-restplus==0.13.0
+flask-swagger-ui==3.20.9
+flask==1.1.1
+gevent==1.5a2             # via geventhttpclient-wheels, locustio
+geventhttpclient-wheels==1.3.1.dev2  # via locustio
+greenlet==0.4.15          # via gevent
+idna==2.8                 # via requests
+importlib-metadata==1.3.0  # via jsonschema, pluggy, pytest
+itsdangerous==1.1.0       # via flask
+jinja2==2.10.3            # via flask
+jsonschema==3.2.0         # via flask-restplus
+locust==0.0
+locustio==0.13.2          # via locust
+markupsafe==1.1.1         # via jinja2
+more-itertools==8.0.2     # via pytest, zipp
+msgpack-python==0.5.6     # via locustio
+git+https://github.com/arajkumar/osa-data-model@setup.py
```

Comment:
why is it referencing a 3rd party project?

Reply:
idea is to reuse data model implementation from @rootAvish., but it wasn't designed as a exportable python module. I forked it and added setup.py to make it installable by pip. I will raise a PR to merge with osa-data-model@master branch.

Reply:
I believe you already osa-data-model under fabric8-analytics, need to use the same here.